### DISCUSSION TITLE
Wholesale Prices through API

### DISCUSSION THREAD

**Discussion by CCEdwards** [ercot/api-specs#56] [Q&A] [answered]
I am having trouble locating an API endpoint for 'Wholesale Prices' that is seen in the ERCOT mobile application. Is that data accessible through an API endpoint at this time? Thanks for the assist.

**Comment by mackermann-ercot** [comment, marked as answer]
Hi CCEdwards, 

The data you are looking for will be under "Settlement Point Prices at Resource Nodes, Hubs and Load Zones" (NP6-905-CD).  You can either search the report archives at: https://data.ercot.com/data-product-archive/NP6-905-CD, or search for specific data: https://data.ercot.com/data-product-details/NP6-905-CD

Please let us know if you need anything else.

Regards,
-mackermann-ercot

**Comment by CCEdwards** [comment]
Thank you mackermann-ercot. That gets me exactly what I need.
